eng/templates/common-tasks.yml

```diff
@@ -0,0 +1,162 @@
+parameters:
+  platform: ''
+
+steps:
+- ${{ if eq(parameters.platform, 'windows') }}:
+  - pwsh: . "$(Build.SourcesDirectory)\src\ensure-symlinks.ps1"
+    displayName: ensure symlinks
+
+  - task: NodeTool@0
+    displayName: Add NodeJS/npm
+    inputs:
+      versionSpec: $(NodeJSVersion)
+
+  - script: |
+      robocopy "eng\resources" "$(Build.SourcesDirectory)\artifacts"
+      :: robocopy return codes are terrible; 1 means files were copied
+      if "%errorlevel%" == "1" exit /b 0
+      exit /b 1
+    displayName: Prevent test directory crawling
+
+  - pwsh: |
+      $testArg = if ($env:SKIPTESTS -ne "true") { "-test" } else { "" }
+      Write-Host "##vso[task.setvariable variable=_TestArgs]$testArg"
+    displayName: Promote variables
+
+  - script: eng\CIBuild.cmd
+      -configuration $(_BuildConfig)
+      -prepareMachine
+      -sign
+      $(_BuildArgs)
+      $(_TestArgs)
+    displayName: Build
+    env:
+      POCKETLOGGER_LOG_PATH: $(PocketLoggerLogPath)
+      TRYDOTNET_PACKAGES_PATH: $(TryDotNetPackagesPath)
+      DOTNET_INTERACTIVE_FRONTEND_NAME: CI
+
+  - pwsh: ./test-retry-runner.ps1 -buildConfig $env:BUILDCONFIG
+    displayName: Test / Blame
+    workingDirectory: $(Build.SourcesDirectory)
+    env:
+      BUILDCONFIG: $(_BuildConfig)
+    condition: ne(variables['SkipTests'], 'true')
+
+  # This is an output artifact in the official yml.
+  # - task: PublishBuildArtifacts@1
+  #   displayName: Publish Test results and Blame dumps
```

Comment:
Why comment this one out?

Reply:
I've added these back in the parent yml. They sadly cannot be de-duped today.